Batch File/Batch File.sublime-syntax

```diff
@@ -101,6 +101,20 @@ contexts:
               - include: command_set_group
               - include: variables
           - include: command_set_group
+      - match: '\s+/[pP]\s+' # SET from standard input
+        push:
+          - match: '(?=\n)'
```

Comment:
This should probably be `'(?=$\n|{{command_terminators}})'`, or maybe a variation that does allow `<`?